drivers/bmp180/bmp180.c

```diff
@@ -193,7 +193,7 @@ static int _read_ut(const bmp180_t *dev, int32_t *output)
         i2c_release(DEV_I2C);
         return -1;
     }
-    *output = ((uint16_t)ut[0] << 8) | ut[1];
+    *output = ((uint16_t)ut[0] << 8) | ((uint16_t)ut[1]);
```

Comment:
Adding a type cast isn't going to change the code. There is already an implicit cast due to the type of the left side of the or.